pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -408,3 +415,17 @@ func SetupLibvirt() error {
 
 	return nil
 }
+
+func convertToLibvirtXML(domspec string) ([]byte, error) {
```

Comment:
can we get a unit test for this

Reply:
@davidvossel I've removed this function since we actually don't need it anymore. Also, there is no need in libvirt-go-xml package for proper XML format. We already have a proper domain spec from the libvirt-go API. I only changed the way XML marshals the domain spec so it will be more readable, and encoded it while calling the logger.